### ISSUE TITLE
Bug: Deadline Openpype cannot find compatible version

### ISSUE THREAD

**Issue by ClementHector** [quadproduction/OpenPype#474]
### Before posting a new ticket, have you looked through the documentation to find an answer?

Yes I have

### Have you looked through the existing tickets to find any related issues ?

Yes I have

### Version

3.15.11-nightly.2

### What platform you are running OpenPype on?

Linux / Centos

### Current Behavior:

The job deadline fails with this error:
```
RuntimeError: Cannot find compatible version available for version 3.15.11-nightly.3 requested by the job. Please add it through plugin configuration in Deadline or install it to configured directory.
```

### Expected Behavior:

The job must be calculated

### What type of bug is it ?

Happens everytime with different configurations

### Which project / workfile / asset / ...

TEST_OP2_PUB_23_7 / Assets / can / model

### Steps To Reproduce:

1. Open Maya scene
2. Publish
3. See the job in deadline

**Comment by ccaillot** [comment]
In order to resolve this issue, I conducted some research and discovered that it's not exactly a problem, but rather a folder hierarchy issue.

Based on my search, there are two solutions available:

- The first solution is simpler but requires creating another Deadline Custom PreJob Plugin. As mentioned in the Openpype documentation (https://openpype.io/docs/dev_deadline/), deleting the OPENPYPE_VERSION environment variable will force the studio settings version and prevent the error message from appearing.

- The second solution is based on this pull request (https://github.com/ynput/OpenPype/pull/4115), which explains how the Deadline plugins work. It seems that the way we set the parent nomenclature is incorrect. Currently, we only manage major and minor versions in our release directory (which is setted in the Deadline OpenPype Plugin). To address this, I created a symlink in the same folder to manage the patch, and this approach resolved the issue.
